src/test/java/org/jabref/logic/importer/fetcher/ISIDOREFetcherTest.java

```diff
@@ -0,0 +1,89 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.util.Optional;
+
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.entry.types.StandardEntryType;
+import org.jabref.testutils.category.FetcherTest;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@FetcherTest
+public class ISIDOREFetcherTest {
+
+    private ISIDOREFetcher fetcher;
+
+    @BeforeEach
+    public void setup() {
+        this.fetcher = new ISIDOREFetcher();
+    }
+
+    @Test
+    public void checkArticle() throws FetcherException {
+        BibEntry expected = new BibEntry(StandardEntryType.Article)
+                .withField(StandardField.TITLE, "Investigating day-to-day variability of transit usage on a multimonth scale with smart card data. A case study in Lyon")
+                .withField(StandardField.AUTHOR, "Oscar Egu and Patrick Bonnel")
+                .withField(StandardField.YEAR, "2020")
+                .withField(StandardField.JOURNAL, "Travel Behaviour and Society")
+                .withField(StandardField.PUBLISHER, "HAL CCSD, Elsevier")
+                .withField(StandardField.DOI, "10.1016/j.tbs.2019.12.003")
+                .withField(StandardField.URL, "https://isidore.science/document/10670/1.hrzlqd");
+
+        Optional<BibEntry> optionalBibEntry = fetcher.performSearchById("https://isidore.science/document/10670/1.hrzlqd");
+        BibEntry actual = optionalBibEntry.get();
+        
+        assertEquals(Optional.of(expected), Optional.of(actual));
+    }
+
+    @Test
+    public void checkArticle2() throws FetcherException {
+        BibEntry expected = new BibEntry(StandardEntryType.Article)
+                .withField(StandardField.TITLE, "“ Anthony B. Atkinson, Inequality – What Can Be Done ? Cambridge (Mass.) Harvard University Press, 2015, XI-384 p. ”")
```

Comment:
The fetcher should remove `“ ` at the beginnig and ` ”` at the end of the title. Reason: JabRef does not track quotes around titles. (I know that the title itself is brap and needs to be improved. But this should be done by the information provider (or is hard work for JabRef))